components.yaml

```diff
@@ -37,7 +37,7 @@ components:
     metadata: v0.47.3
   ovs-cni:
     url: https://github.com/k8snetworkplumbingwg/ovs-cni
-    commit: 5c85bb78f5475fe5013d577dca5f5f35d91dc298
+    commit: 43e26dd1449f3867f057d74c01125543541aed95
     branch: release-0.19
     update-policy: tagged
-    metadata: v0.19.2
+    metadata: v0.19.3
```

Comment:
@RamLavi 
something is weird, quay.io/kubevirt/ovs-cni-plugin has indeed v0.19.3
and this bump should be correct

however it said conflict
and if i run locally i see that it doesnt change anything, it still tries to use v0.19.2 

wdyt ?